helm/dex-app/values.yaml

```diff
@@ -59,7 +59,7 @@ oidc:
     connectorName: ""
     connectorConfig: ""
     write_all_group: ""
-    connectors:
+    connectors: []
```

Comment:
I had to make this change for the schema validation to read this as an array. However, I will need to make sure this does not affect the current templating.

Reply:
:+1: 
Please share your finding after checking on existing templating.

Reply:
Works just fine. I deployed to `gauss` successfully and the customer connector (not using `connectors` field) was templated correctly.